lib/initialize.js

```diff
@@ -0,0 +1,99 @@
+const path = require('path')
+const rimraf = require('rimraf')
+const fs = require("fs")
+const constants = require("./constants")
+const {createCustomFileAccessor} = require("./config")
+const {fileFuncs} = require("./file-operations");
+
+//The two locations parsec app data folder might exist
+const parsecDataLocationDefault = path.join(constants.appDataPath,'..','Parsec')
+const parsecDataLocationOther = "C:\\ProgramData\\Parsec"
+
+//flag for parsecd.exe found
+
+
+//flag for parsec app data location found
+const flags = {
+    parsecdFound : 0,
+    parsecDataLocationFound : 0
+}
+
+
+
+let parsecDataLocation = ""
+
+//event which will be run if config or data are modified
+const onConfigChanged = []
```

Comment:
no need to define this globally as its only used in `intialize` function scope

Reply:
done